Release note MBI 21.04 (#1313)
centreon/centreon-documentation: versioned_docs/version-21.04/releases/centreon-commercial-extensions.md

## Centreon MBI

### 21.04.3

`February 18 2022`

#### Security fixes

- Log4J MBI upgrade to 2.17.1

#### Improvements

- Add new column “reported duration” in report event list

> - Replace 'switch' with 'if' to accomodate RHEL 8 deployement

Reviewer: ```suggestion
- Replace 'switch' with 'if' to accommodate RHEL 8 deployment
```

> - Compatibility PHP 7.4 (20.10 / 21.04) for MBI

Reviewer: ```suggestion
- Compatibility with PHP 7.4 (20.10 / 21.04) for MBI
```

#### Bugfixes

- Fixed: New line character missing from last line in MBI back up script preventing the script to execute
- Fixed: Can't find init MySQL script if mariadb only configured for systemd
- Fixed: Report job failed due to incompatibility with the Jsch library
- Fixed: A service associated with several categories of services will be present several times on the reports (ex Hostgroups-Service-Current-Events)
- Fixed: Notification for administrators not working on fail
- Fixed: Hostgroups-Incidents-1 English translation issue (”Maintainability”)
- Fixed: Availability on host or hostgroup, rounding issue

> - Fixed: Report BA-Availability-1 - Events when KPIs are in downtime should not be listed

Reviewer: ```suggestion
- Fixed: Report BA-Availability-1 - When services are in downtime, events should not be listed
```
